src/db/migration-helpers.ts

```diff
@@ -12,29 +12,14 @@ type NftFactoryAddress = {
 }
 
 export const addPlatform = async (knex: Knex, platform: MusicPlatform) => {
-  await knex.raw(`ALTER TABLE platforms drop constraint "platforms_type_check"`);
+  await knex.raw(`ALTER TABLE "${Table.platforms}" drop constraint "platforms_type_check"`);
```

Comment:
@Geimaj actually is this still platforms_type_check or will it become like raw_platforms_type_check?

Reply:
hm platforms_type_check is just the name of the constraint, so should be fine to leave as is i think?